src/translations/en.json

```diff
@@ -2799,24 +2801,27 @@
             "inclusion_finished": "The device has been added.",
             "view_device": "View Device",
             "interview_started": "The device is being interviewed. This may take some time.",
-            "interview_failed": "The device interview failed. Additional information may be available in the logs.",
-            "security_classes": {
-              "S2_Unauthenticated": {
-                "title": "S2 Unauthenticated",
-                "description": "Like S2 Authenticated, but without verification that the correct device is included"
-              },
-              "S2_Authenticated": {
-                "title": "S2 Authenticated",
-                "description": "Example: Lighting, Sensors and Security Systems"
-              },
-              "S2_AccessControl": {
-                "title": "S2 Access Control",
-                "description": "Example: Door Locks and Garage Doors"
-              },
-              "S0_Legacy": {
-                "title": "S0 Legacy",
-                "description": "Example: Legacy Door Locks without S2 support"
-              }
+            "interview_failed": "The device interview failed. Additional information may be available in the logs."
+          },
+          "security_classes": {
```

Comment:
This will need an update to Lokalise after it is merged.

Reply:
do I need to do anything about that? and do you think I should have kept it where it was instead?

Reply:
No, it is fine :-) And I'll handle it